webapi/src/main/scala/org/knora/webapi/responders/v2/ResourcesResponderV2.scala

```diff
@@ -916,24 +934,39 @@ class ResourcesResponderV2(responderData: ResponderData) extends ResponderWithSt
       * @param preview      `true` if a preview of the resource is requested.
       * @param versionDate  if defined, requests the state of the resources at the specified time in the past.
       *                     Cannot be used in conjunction with `preview`.
+      * @param targetSchema the target API schema.
+      * @param schemaOptions the schema options submitted with the request.
       * @return a map of resource IRIs to RDF data.
       */
     private def getResourcesFromTriplestore(resourceIris: Seq[IRI],
                                             preview: Boolean,
                                             propertyIri: Option[SmartIri],
                                             versionDate: Option[Instant],
+                                            targetSchema: ApiV2Schema,
+                                            schemaOptions: Set[SchemaOption],
                                             requestingUser: UserADM): Future[Map[IRI, ResourceWithValueRdfData]] = {
 
         // eliminate duplicate Iris
         val resourceIrisDistinct: Seq[IRI] = resourceIris.distinct
 
+        // If we're supposed to query standoff, get the indexes delimiting the first page of standoff. (Subsequent
+        // pages, if any, will be queried separately.)
+        val (maybeStandoffMinStartIndex: Option[Int], maybeStandoffMaxStartIndex: Option[Int]) = StandoffTagUtilV2.getStandoffMinAndMaxStartIndexesForTextValueQuery(
```

Comment:
Is this necessary? Isn't this already determined in `getResourcesV2 `:

`val queryStandoff = SchemaOptions.queryStandoffWithTextValues(targetSchema = targetSchema, schemaOptions = schemaOptions)`

Reply:
You're right, I'll fix that.

Reply:
Done in fe74ba6.